server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandlerHelper.java

```diff
@@ -49,6 +50,28 @@ public static Boolean checkMembership(ReadContext readContext, String userId, St
     return (row != null) ? true : false;
   }
 
+  /**
+  * Returns a Boolean that indicates whether or not a person is the owner of a club.
+  * This method creates a Struct that holds a single row from a database read row transacation.
+  * It returns false if the Struct is null, indicating the membership does not exist.
+  * It returns true if the Struct is not null, indicating the membership does exist.
+  *
+  * @param  readContext  the context for an attempt to perform a transaction
+  * @param  userId       the user ID string of the user we are checking is in a club
+  * @param  clubId       the club ID string of the club we are checking the user is in
+  * @return              the Boolean true or false representing if the membership exists or not
+  */
+  public static Boolean checkOwnership(ReadContext readContext, String userId, String clubId) {
+    Struct row =
+          readContext
+            .readRow(
+              "Memberships",
+              Key.of(userId, clubId),
+              Arrays.asList("membershipType"));
+    return ((row != null) && (row.getLong(/** index = **/0) == MembershipConstants.OWNER))
+            ? true : false;
```

Comment:
What does this ternary operator do? From a naive perspective it seems to be redundant but I could be missing something.

Reply:
So, because memberships table can hold memberships of type owner and member, the `(row != null) && (row.getLong(/** index = **/0) == MembershipConstants.OWNER)` checks that the result of the strong read is not null (there is a membership which exists), and then checks that the membership is of type owner. If both are true, it returns true, else false. This is where `checkMembership` differs from `checkOwnership` -- this first one is checking whether their membership is in the database table (could be member or owner). `checkOwnership` is actually not called anywhere except in the tests to verify that that adding an ownership works and that it differentiates between owners and members.

Reply:
I think I am going to change `checkMembership` to be `checkAnyMembership` since that name reflects what the method is doing better.

Reply:
It seems as though the `((row != null) && (row.getLong(/** index = **/0) == MembershipConstants.OWNER))` simplifies down to a boolean itself. Thus, the ternary operator sees `boolean ? true : false`, which is a little redundant because you'd get the same result by just returning the boolean.

Reply:
Oh I see what you meant! I'll change that up now.